packages/components/package.json

```diff
@@ -30,7 +30,7 @@
     "deploy:docs": "storybook-to-ghpages --ci",
     "prepack": "npm run build && cp ./package.json ./tailwind.config.js dist/",
     "snapshot": "percy-storybook --widths=320,1280",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -p 6008",
```

Comment:
why not `6007`? just asking b/c that's what I get prompted with whenever 6006 isn't available

Reply:
I should probably go with a more arbitrary value, but basically because 6007 is the next one, when I was messing with it, it was also already taken, so just naively spacing it apart more.

Reply:
oh lol 😅